pkg/library/flatten/flatten.go

```diff
@@ -29,12 +30,21 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	// DWTSupportedNamespacesAnnotation defines additional namespaces from which a DevWorkspace can import a DevWorkspaceTemplate.
+	// Options are:
+	// - '*': allow importing by all DevWorkspaces on the cluster
+	// - 'namespaceA,namespaceB,namespaceC': Allow importing by DevWorkspaces in list of specific namespaces
+	// If the annotation does not exist or is empty, only DevWorkspaces in the same namespace as the template can reference it.
+	DWTSupportedNamespacesAnnotation = "controller.devfile.io/supported-namespaces"
```

Comment:
suported-namespace is pretty common and it's not clear which level of support is meant to be here.
Maybe `controller.devfile.io/allow-from-namespaces`, `controller.devfile.io/accessible-from-namespaces` would be a bit clear but I'm not sure, I don't like them much as well.

> DWTSupportedNamespacesAnnotation defines additional namespaces from which a DevWorkspace can import a DevWorkspaceTemplate.

`controller.devfile.io/allow-import-from` or `controller.devfile.io/allow-references-from`?

Reply:
@amisevsk @JPinkney I just flushed with all I managed to come to, feel free to share your opinion or propose your variants. I'm actually OK with any )

Reply:
I went with `controller.devfile.io/allow-import-from`